app/src/main/java/net/squanchy/typeface/TypefaceDelegate.kt

```diff
@@ -0,0 +1,32 @@
+package net.squanchy.typeface
+
+import android.graphics.Paint
+import android.graphics.Typeface
+import net.squanchy.extensions.whenTrue
+
+class TypefaceDelegate(private val newTypeface: Typeface) {
+
+    private companion object {
+        @SuppressWarnings("MagicNumber")
+        const val TEXT_SKEW_X = -.25f
```

Comment:
This one is seen by Detekt as `MagicNumber`, but it's a constant, maybe I'm declaring it in the wrong way

Reply:
Looks like a bug in Detekt, please file an issue --> https://github.com/arturbosch/detekt/issues

Reply:
It looks like there's already an open issue, exactly for this: https://github.com/arturbosch/detekt/issues/467

Reply:
did you try putting it outside of the class?

Reply:
@fourlastor this is only relevant to the class, tho. I would keep it private. What do you think?

Reply:
That doesn't mention constants at all, it's an unrelated issue

Reply:
we could have it file private probably, shouldn't be too different

Reply:
Doesn't seem right to just put it outside, but we can definitely do it

Reply:
Should definitely be private, yes. I wouldn't put it outside thou.

Reply:
@rock3r it doesn't mention `const`, but it does mention `private val`... I think the problem here is more with `val` position than with `const`, which should only be a compile time modifier (https://kotlinlang.org/docs/reference/properties.html#compile-time-constants)